.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+# Summary | Résumé
+
+> 1-3 sentence description of the changed you're proposing.
+> Please include the GitHub issue number or relevant acceptance criteria.
+> For any UI changes please include screenshots at different breakpoints.
+
+---
+
+> Description en 1 à 3 phrases de la modification proposée, avec un lien vers le
+> problème (« issue ») GitHub ou la fiche Trello, le cas échéant.
+
+# Test instructions | Instructions pour tester la modification
+
+> Sequential steps (1., 2., 3., ...) that describe how to test this change. This
+> will help a developer test things out without too much detective work. Also,
+> include any environmental setup steps that aren't in the normal README steps
+> and/or any time-based elements that this requires.
+
+---
+
+> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de tester la
+> modification. Elles aideront les développeurs à faire des tests sans avoir à
+> jouer au détective. Veuillez aussi inclure toutes les étapes de configuration
+> de l’environnement qui ne font pas partie des étapes normales dans le fichier
+> README et tout élément temporel requis.
+
+# Help requested | Aide requise
```

Comment:
Wondering how useful this section really is.  I would hope that once a PR is ready for review all the 'requesting help' has already been completed.